### ISSUE TITLE
xfe not found

### ISSUE THREAD

**Issue by nastybuttler322** [deadhead420/anarchy-linux#605]
downloading anarchy.db...
:: There are 5 members in group i3:
:: Repository community
   1) i3-gaps  2) i3-wm  3) i3blocks  4) i3lock  5) i3status

Enter a selection (default=all): error: target not found: xfe

==> ERROR: Failed to install packages to new root
2018-01-09 15:39 : Install failed: please report to developer

**Comment by nastybuttler322** [comment]
If this is not able to be duplicated, than this is another UEFI problem. I have 2 computers, and I was initially trying to install this with UEFI on this machine since it's a lot newer. But I don't remember if this was during a UEFI installation or now. I did a lot of trial-and-error to get this set up.

I might do another fresh install on this machine because I have messed with a lot of settings from advice through google searches on getting the wifi set up, and now whatever advice may have worked may not at this point.
